### DISCUSSION TITLE
Image, Lists and other Components inside Table

### DISCUSSION THREAD

**Discussion by MuhammadAhsan252** [codex-team/editor.js#2619] [Ideas]
Currently we don't have the option to add different components inside Table such  as Images, lists etc. I have seen some editors like CKeditor and Rich Text Editor provided by SyncFusion where they are supporting this functionality. So, I wonder if this would be a useful addition in editor js as well.

Example:
<img width="1082" alt="Screenshot 2024-02-16 at 4 51 28 PM" src="https://github.com/codex-team/editor.js/assets/59654598/12bf0210-146b-4954-a185-65141646a94d">

**Comment by mikkel1156** [comment]
This to me also seems like a useful feature. Kind of like how block tunes are enabled, can enable block editing within a block.

Only way I can imagine it right now is embedding editorjs within itself? But seems like a silly workaround.

Any pointers on if this is planned or some pointers for anyone wanting to try a hand at a PR?

**Comment by kamrul-CSE-official** [comment]
Yes I also face this issues but not find any sulation
